src/ucar/unidata/apps/lead/tb0Catalog.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <catalog name="LEAD TestBed IDV Data" version="0.6"
-xmlns="http://www.unidata.ucar.edu/thredds"
```

Comment:
Wow! This XML Namespace must be very old. We've used the www.unidata.ucar.edu/namespaces/* URIs for a very long time.